modules/xfem/src/constraints/XFEMSingleVariableConstraint.C

```diff
@@ -25,24 +26,32 @@ InputParameters
 validParams<XFEMSingleVariableConstraint>()
 {
   InputParameters params = validParams<ElemElemConstraint>();
-  params.addParam<Real>("alpha", 100, "Stablization parameter in Nitsche's formulation.");
+  params.addParam<Real>("alpha",
+                        100,
+                        "Stabilization parameter in Nitsche's formulation and penalty factor "
+                        "in the Penalty Method. In Nitsche's formulation this should be as "
+                        "small as possible while the method is still stable; while in the "
+                        "Penalty Method you want this to be quite large (e.g. 1e6).");
   params.addParam<Real>("jump", 0, "Jump at the interface.");
```

Comment:
Can you also change 'jump' from Real to Function?

Reply:
Yes, I'll do that.